Description: track default symlinks, instead of symlinking
Forwarded: not-needed
--- a/cmd/zed/zed.d/Makefile.am
+++ b/cmd/zed/zed.d/Makefile.am
@@ -50,7 +50,7 @@
 	set -x; for f in $(zedconfdefaults); do \
 	  [ -f "$(DESTDIR)$(zedconfdir)/$${f}" ] ||\
 	    [ -L "$(DESTDIR)$(zedconfdir)/$${f}" ] || \
-	    $(LN_S) "$(zedexecdir)/$${f}" "$(DESTDIR)$(zedconfdir)"; \
+	    echo "$${f}" >> "$(DESTDIR)$(zedexecdir)/DEFAULT-ENABLED" ; \
 	done
 
 SHELLCHECKSCRIPTS += $(dist_zedconf_DATA) $(dist_zedexec_SCRIPTS) $(nodist_zedexec_SCRIPTS)
