Description: relocate the executable path
Forwarded: no need
Index: zfs/etc/systemd/system/zfs-volume-wait.service.in
===================================================================
--- zfs.orig/etc/systemd/system/zfs-volume-wait.service.in
+++ zfs/etc/systemd/system/zfs-volume-wait.service.in
@@ -9,7 +9,7 @@ ConditionPathIsDirectory=/sys/module/zfs
 Type=oneshot
 RemainAfterExit=yes
 EnvironmentFile=-@initconfdir@/zfs
-ExecStart=@bindir@/zvol_wait
+ExecStart=/sbin/zvol_wait
 
 [Install]
 WantedBy=zfs-volumes.target
