Description: relocate the executable path
Forwarded: not-needed
Index: zfs-linux/etc/systemd/system/zfs-volume-wait.service.in
===================================================================
--- zfs-linux.orig/etc/systemd/system/zfs-volume-wait.service.in
+++ zfs-linux/etc/systemd/system/zfs-volume-wait.service.in
@@ -9,7 +9,7 @@ ConditionPathIsDirectory=/sys/module/zfs
 Type=oneshot
 RemainAfterExit=yes
 EnvironmentFile=-@initconfdir@/zfs
-ExecStart=@bindir@/zvol_wait
+ExecStart=@sbindir@/zvol_wait
 
 [Install]
 WantedBy=zfs-volumes.target
