Description: force libtool to produce verbose output
Author: Mo Zhou
Forwarded: not-needed
Index: zfs-linux/config/Rules.am
===================================================================
--- zfs-linux.orig/config/Rules.am
+++ zfs-linux/config/Rules.am
@@ -12,7 +12,7 @@ AM_CPPFLAGS = \
 	-I$(top_srcdir)/lib/libspl/include \
 	-I$(top_srcdir)/lib/libspl/include/os/@ac_system_l@
 
-AM_LIBTOOLFLAGS = --silent
+AM_LIBTOOLFLAGS =
 
 AM_CFLAGS  = -std=gnu99 -Wall -Wextra -Wstrict-prototypes -Wmissing-prototypes -Wwrite-strings -Wno-sign-compare -Wno-missing-field-initializers
 AM_CFLAGS += -fno-strict-aliasing
