diff --git a/etc/init.d/zfs-zed.in b/etc/init.d/zfs-zed.in
index d0086eef..dd735638 100644
--- a/etc/init.d/zfs-zed.in
+++ b/etc/init.d/zfs-zed.in
@@ -8,8 +8,8 @@
 #
 ### BEGIN INIT INFO
 # Provides:          zfs-zed
-# Required-Start:    zfs-mount
-# Required-Stop:     zfs-mount
+# Required-Start:    $local_fs zfs-mount
+# Required-Stop:     $local_fs zfs-mount
 # Default-Start:     2 3 4 5
 # Default-Stop:      0 1 6
 # X-Stop-After:      zfs-share
